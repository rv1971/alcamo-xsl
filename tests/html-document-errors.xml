<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"
      xml:id="xslt"
      version="1.0"
      exclude-result-srefixes="a">
    <xsl:import href="../xsl/html-document.xsl"/>

    <xsl:param name="a:errorReportingChannels" select="'output'"/>

    <xsl:template match="/">
      <xsl:call-template name="a:report-errors"/>
    </xsl:template>

    <xsl:template name="a:collect-errors">
      <p>Lorem ipsum.</p>
    </xsl:template>

  </xsl:stylesheet>
</a:doc>
