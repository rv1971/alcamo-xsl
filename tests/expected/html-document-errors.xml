<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" class="error">
  <p xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">Lorem ipsum.</p>
</div>
